bluesky_widgets/models/plot_builders.py

```diff
@@ -203,6 +207,13 @@ def set_figure(event):
         self.add_run = self._run_manager.add_run
         self.discard_run = self._run_manager.discard_run
 
+        self.ys.events.added.connect(self._add_ys)
+        self.ys.events.removed.connect(self._remove_ys)
+
+    def _map_ys_to_artists(self, artist, y):
```

Comment:
Does this method buy us anything? Can the callers just touch `self._ys_to_artist` directly? If we anticipate that more logic will be added here in time, that would be a good reason to keep it, but I don't see that yet from a quick read.